skypy/halos/tests/test_sampler.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import colossus as colossus
+from colossus.lss import mass_function
+from scipy import integrate
+from scipy.stats import kstest
+import pytest
+
+
+@pytest.mark.flaky
+def test_colossus_mass_sampler():
+    from astropy.cosmology import WMAP9
+    from skypy.halos.sampler import colossus_mass_sampler
+    m_min = 1e+12
+    sample = colossus_mass_sampler(redshift=0.1, model='despali16',
+                                   mdef='500c', m_min=m_min, m_max=1e+16,
+                                   cosmology=WMAP9, sigma8=0.8200, ns=0.9608,
+                                   size=100, resolution=1000)
+    assert np.all(sample >= m_min)
```

Comment:
```suggestion
    m_min, m_max, size = 1e+12, 1e+16, 100
    sample = colossus_mass_sampler(redshift=0.1, model='despali16',
                                   mdef='500c', m_min=m_min, m_max=m_max,
                                   cosmology=WMAP9, sigma8=0.8200, ns=0.9608,
                                   size=size, resolution=1000)
    assert np.all(sample >= m_min)
    assert np.all(sample <= m_max)
    assert np.shape(sample) == (size,)
```
Test that samples are not greater than m_max and that the correct number of samples are returned

Reply:
Thanks for these suggested tests. Will add them to the code!